### PR TITLE
Authentication part finished.

### PR DESCRIPTION
**Pull request by starkshaw**

### CODE REVIEW

File: app/assets/stylesheets/Bars.css.sass [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.navbar-brand
```
Comment (danielpaul) [inline app/assets/stylesheets/Bars.css.sass]: You don't need to have `.css.sass`. Just `.sass` is fine. Also usually always a good idea to lowercase filenames.

File: app/assets/stylesheets/application.css_backup [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*
```
Comment (danielpaul) [inline app/assets/stylesheets/application.css_backup]: This file should be deleted i'm guessing?

Comment (starkshaw) [inline app/assets/stylesheets/application.css_backup]: Yes

**Comment by danielpaul** [comment]
Good to merge :+1:

**Pull request merged**